doc/modules/model_evaluation.rst

```diff
@@ -1923,41 +1923,62 @@ set [0,1] has an error::
 Brier score loss
 ----------------
 
-The :func:`brier_score_loss` function computes the
-`Brier score <https://en.wikipedia.org/wiki/Brier_score>`_
-for binary classes [Brier1950]_. Quoting Wikipedia:
+The :func:`brier_score_loss` function computes the `Brier score
+<https://en.wikipedia.org/wiki/Brier_score>`_ for binary or multiclass
+probabilistic predictions. Quoting Wikipedia:
 
     "The Brier score is a proper score function that measures the accuracy of
     probabilistic predictions. It is applicable to tasks in which predictions
     must assign probabilities to a set of mutually exclusive discrete outcomes."
 
-This function returns the mean squared error of the actual outcome
-:math:`y \in \{0,1\}` and the predicted probability estimate
-:math:`p = \operatorname{Pr}(y = 1)` (:term:`predict_proba`) as outputted by:
+Let the true labels for a set of :math:`N` samples be encoded as a 1-of-K binary
```

Comment:
We decided to try refraining from using "samples" when referring to "data points"/"observations"/"records" in our English doc as a "sample" could represent a full dataset (e.g. the training set) and the reader could get confused as a result.

```suggestion
Let the true labels for a set of :math:`N` data points be encoded as a 1-of-K binary
```

Note that we don't plant to rename `n_samples` to `n_data_points` for the sake of backward compatibilities.